### DISCUSSION TITLE
Discussion about distributed storage and training in PyG

### DISCUSSION THREAD

**Discussion by eedalong** [pyg-team/pytorch_geometric#2702] [Ideas]
Hi, What's your plan about distributed training in PyG? I think we can implement this feature together

**Comment by rusty1s** [comment]
Thanks for your interest. There is no concrete plan yet about distributed graph storage (interested to hear your thoughts on this one), but distributed training is fully supported (either via standard PyTorch or PyTorch Lightning).
